### PR TITLE
update for 2.6.4

### PR DESCRIPTION
**Pull request by Acogkr**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -3,11 +3,9 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 2
```
Comment (btk5h) [inline .editorconfig:6]: Can you revert these .editorconfig changes and all related whitespace changes?

File: build.gradle [hunk 2]
```diff
@@ -1,45 +1,60 @@
...
+
+//task buildReadme(type: Javadoc) {
+//  source = sourceSets.main.allJava
+//  classpath = sourceSets.main.compileClasspath
+//  destinationDir = projectDir
+//  options.  = [file('tools/skriptdoclet.jar')]
+//  options.doclet = 'com.btk5h.skriptdoclet.SkriptDoclet'
+//  options.addStringOption('file', 'README.md')
+//  options.addStringOption('markdown', '-quiet')
+//}
```
Comment (btk5h) [inline build.gradle:60]: Why is this left commented in the `build.gradle`?

File: src/main/java/com/btk5h/skriptdb/skript/ExprDataSource.java [hunk 3]
```diff
@@ -29,69 +28,74 @@
...
+    //    public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed,
+    //                        SkriptParser.ParseResult parseResult) {
+    //        url = exprs[0];
+    //        maxLifetime = exprs[1];
+    //        return true;
+    //    }
+    @Override
+    public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, SkriptParser.ParseResult parseResult) {
+        return false;
+    }
```
Comment (btk5h) [inline src/main/java/com/btk5h/skriptdb/skript/ExprDataSource.java:100]: Why is this commented? Why was this changed to return false?

**Review by btk5h — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution. I'd prefer if you kept PRs focused only on the changes required to implement the feature you are working on. There are quite a few arbitrary code style/whitespace changes that I don't think need to be in this PR.

File: .editorconfig (hunk 1 above)
Comment (Acogkr) [reply .editorconfig:6]: it's possible. What is the reason for reverting white space?

**Comment by Acogkr** [comment]
If so, it's okay to cancel the PR. I thought it was a discontinued feature, so I created a new one at random. If you want, I will upload a new version and it. Sorry.

**Comment by btk5h** [comment]
You're correct in that I'm not really actively maintaining this project right now, you're free to maintain your own fork though and I'll happily link to it in the README
